sql/audit_gm.sql

```diff
@@ -0,0 +1,20 @@
+-- MySQL dump 10.13  Distrib 5.7.12, for Win64 (x86_64)
+--
+-- Host: localhost    Database: dspdb
+-- ------------------------------------------------------
+-- Server version	5.5.5-10.0.20-MariaDB
+
+--
+-- Table structure for table `audit_gm`
+--
+
+DROP TABLE IF EXISTS `audit_gm`;
+CREATE TABLE `audit_gm` (
+  `date_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
In your SQL in the command handler you insert a time stamp, I don't see where a timestamp wouldn't be inserted, so wouldn't the default not be necessary?

Reply:
iunno we do that with out other tables to except they default to all 0's which is deprecated